File: dart/lib/model/v3_fare_estimate_response.dart
part of swagger.api;

class V3FareEstimateResponse {
  
  V3FareEstimateResultStatus fareEstimateResultStatus = null;

  V3FareEstimateResult fareEstimateResult = null;

  V3FareEstimateResponse();

  @override
  String toString() {
    return 'V3FareEstimateResponse[fareEstimateResultStatus=$fareEstimateResultStatus, fareEstimateResult=$fareEstimateResult, ]';
  }

  V3FareEstimateResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    fareEstimateResultStatus = new V3FareEstimateResultStatus.fromJson(json['FareEstimateResultStatus']);
    fareEstimateResult = new V3FareEstimateResult.fromJson(json['FareEstimateResult']);
  }

  Map<String, dynamic> toJson() {
    return {
      'FareEstimateResultStatus': fareEstimateResultStatus,
      'FareEstimateResult': fareEstimateResult
     };
  }

  static List<V3FareEstimateResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3FareEstimateResponse>() : json.map((value) => new V3FareEstimateResponse.fromJson(value)).toList();
  }

  static Map<String, V3FareEstimateResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3FareEstimateResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3FareEstimateResponse.fromJson(value));
    }
    return map;
  }
}

File: csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/V3FareEstimateResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class V3FareEstimateResponse {
    /// <summary>
    /// Gets or Sets FareEstimateResultStatus
    /// </summary>
    [DataMember(Name="FareEstimateResultStatus", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "FareEstimateResultStatus")]
    public V3FareEstimateResultStatus FareEstimateResultStatus { get; set; }

    /// <summary>
    /// Gets or Sets FareEstimateResult
    /// </summary>
    [DataMember(Name="FareEstimateResult", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "FareEstimateResult")]
    public V3FareEstimateResult FareEstimateResult { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class V3FareEstimateResponse {\n");
      sb.Append("  FareEstimateResultStatus: ").Append(FareEstimateResultStatus).Append("\n");
      sb.Append("  FareEstimateResult: ").Append(FareEstimateResult).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
